pkg/provider/handlers/functions.go

```diff
@@ -61,7 +62,7 @@ func GetFunction(client *containerd.Client, name string) (Function, error) {
 				replicas = 1
 				f.pid = task.Pid()
 				// Get container IP address
-				ip, _ := GetIPfromPID(int(task.Pid()))
+				ip, _ := cninetwork.GetIPfromPID(int(task.Pid()))
```

Comment:
We can't swallow errors like this, so since you're working on this line. Please can you handle it?

Reply:
Done!